lib/handle-branch-status.js

```diff
@@ -148,7 +148,7 @@ module.exports = async function (
     owner,
     repo,
     number,
-    body: `## Version **${version}** just got published. \nYour tests are passing again with this version. [Explicitly upgrade to this version 🚀](${compareURL}) ${bodyDetails}`
+    body: `## Version **${version}** just got published. \nYour tests ${branchDoc.group ? ` for group **${branchDoc.group}** ` : ''}are passing again with this version. [Explicitly upgrade ${branchDoc.group ? `**${branchDoc.group}** ` : ''}to this version 🚀](${compareURL}) ${bodyDetails}`
```

Comment:
Double space generate here if there’s a group: `Your tests  for group` (between test and for).